Add local search

bcl opened the pull request:
This uses https://github.com/cmfcmf/docusaurus-search-local to add a local search feature with no dependency on outside services.

dvdhrm commented on _collections/_news/2020-06-04-release-osbuild-composer-14.md:
```diff
@@ -0,0 +1,60 @@
+issues, yesterday our team decided to roll-out an off-schedule bug-fix
+release. Say hi to osbuild-composer 14.
+
+The most important news is: AWS upload support got several fixes making it
+more stable and the internal store is now error-prone to manifests with
+an old schema. Also, we dropped support for i686 architecture. Building
+images for it was never supported and we now no longer provide i686 RPMs.
```
You sure your use of `error-prone` is correct here? Did you mean `error-proof`, or `resilient`?

ondrejbudai replied:
good catch, I changed the wording.

brlane-rht committed:
Add local search

This uses https://github.com/cmfcmf/docusaurus-search-local to add a
local search feature with no dependency on outside services.

achilleas-k reviewed (approved):
COOOOOL!
